src/components/evidence-tag.tsx

```diff
@@ -8,18 +15,45 @@ import '../styles/components/evidence-tag.scss';
 
 const size = 12;
 
-const EvidenceTag = ({ label, title, className, iconComponent, children }) => {
+type EvidenceTagProps = {
+  /**
+   * Displayed on the tag
+   */
+  label: string;
```

Comment:
```suggestion
  label: ReactNode;
```

Reply:
I think we want to be quite restrictive about that and only allow strings.